macaroon-minting.xml

```diff
@@ -52,7 +52,7 @@
       was issued.</t>
 
       <t>This document describes a simple API that allows a client to
-      request a macaroon be minted that captures the client's
+      request a macaroon to be minted that encodes the client's
```

Comment:
I'm not sure the extra "to" is needed (or even is correct) here, but I can rephrase the sentence if it is confusing.